packages/web/scripts/security.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+tempFile=$(mktemp)
+cat > "${tempFile}" <<EOF
+
+        add_header X-Frame-Options "SAMEORIGIN";
+        add_header X-Content-Type-Options "nosniff";
+        add_header Referrer-Policy "strict-origin-when-cross-origin";
+        add_header Content-Security-Policy "default-src 'self'; script-src 'self'; style-src 'self' 'unsafe-inline'; frame-ancestors 'self'; connect-src 'self' http://${API_DOMAIN} https://${API_DOMAIN} ws://${API_DOMAIN} wss://${API_DOMAIN} blob:;";
```

Comment:
`script-src` and `frame-ancestors` are redundant to `default-src`

Reply:
what is the `API_DOMAIN`? I don't think we are passing that there

Reply:
I'll remove it for now, it's only needed if they deploy the API on a different domain than the dashboard.
It's harder to parse the hostname from REACT_APP_API_SERVER_ENDPOINT instead of adding a new API_DOMAIN url, but I don't want to repeat the logic either.. will see what their input is